antidox/perspective.py

```diff
@@ -58,14 +57,11 @@ def dlp_request(dlp, apikey_data, comment):
   "includeQuote":True
   }
   }
-  #if dlp_request["inspectConfig"]["minLikelihood"] == "POSSIBLE" :
-  #  print ("this is personal info")
   dlp_response = dlp.projects().content().inspect(body = dlp_request, parent = 'projects/' + apikey_data['project_number']).execute()
   return dlp_response
-  #print (json.dumps(dlp_response, indent=2))
-  #print (dlp_request.keys())
 
 
+#Checking and returning only for comments that are likely or very likely to contain PII
```

Comment:
Can you make all lines 80 characters long or less?

In Sublime Text you can easily see what lines are above 80 characters by selecting from the View menu --> Ruler --> 80.  In this specific case you can just make a 2 line comment.

You can read more about this rule and the few exceptions here: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length